README.md

```diff
@@ -116,13 +116,13 @@ The software is provided as is under no guarantee.
 I will update both the repo and this readme as I go during following days (date of writing this is 26th of February 2022, third day into russian invasion into Ukraine)
 
 Synflood implementation is taken from https://github.com/bilalcaliskan/syn-flood and slightly patched. I couldn't just import the package as all the functionality code was in an internal package preventing import into other modules. Will figure it out better later (sorry to the owner).
-## How to install
+# How to install
```

Comment:
```suggestion
## How to install
```

This is the proper markdown formatting according to my linter (there can be only one top level heading in a document)

Reply:
I'll add and apply other suggestions based on that (just found out it's possible to do this in github)

Reply:
@Arriven Perhaps I am using markdown a bit wrongly, but I see no point having only one top-level heading. I used to separate different categories by top-level heading and then go down accordingly